server/schedule/operator_controller.go

```diff
@@ -316,3 +318,100 @@ func (oc *OperatorController) OperatorCount(mask OperatorKind) uint64 {
 	}
 	return total
 }
+
+// OpInfluenceFilterOption used to filter opInfluence.
+type OpInfluenceFilterOption func(opInfluence OpInfluence)
+
+// GetOpInfluence gets OpInfluence.
+func (oc *OperatorController) GetOpInfluence(opts ...OpInfluenceFilterOption) OpInfluence {
```

Comment:
Maybe we can use `Cluster` as a parameter, and use the cluster to distinguish operators.